programs/pkey/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+set(libs
```

Comment:
This is not correct - `libs` here is just a variable, and is not used - however the `target_link_libraries()` here already includes `${mbedcrypto_target}` (The usual thing in the programs CMakeLists.txt is to set `$libs` and then use it in the `target_link_libraries()` call)

Also, I think adding just `mbedtls` to `target_link_libraries()` would potentially cause it to link against the system installed mbedtls if it existed.